samples/scala-value-entity-customer-registry/build.sbt

```diff
@@ -28,20 +28,3 @@ Test / logBuffered := false
 
 run / fork := false
 Global / cancelable := false // ctrl-c
-
-val LogbackVersion = "1.2.3"
-val LogbackContribVersion = "0.1.5"
-val logback = "ch.qos.logback" % "logback-classic" % LogbackVersion
-val logbackJson = "ch.qos.logback.contrib" % "logback-json-classic" % LogbackContribVersion
-val logbackJackson = "ch.qos.logback.contrib" % "logback-jackson" % LogbackContribVersion
-
-// FIXME include these dependencies via the AkkaserverlessPlugin
-val AkkaServerlessSdkVersion = System.getProperty("akkaserverless-sdk.version", "0.7.2")
-libraryDependencies ++= Seq(
-  logback,
-  logbackJson,
-  logbackJackson,
-  "org.scalatest" %% "scalatest" % "3.2.7" % Test,
```

Comment:
ScalaTest must remain here, if you agree with https://github.com/lightbend/akkaserverless-java-sdk/pull/508#discussion_r717260609